cmd/argo/commands/copy_artifacts.go

```diff
@@ -0,0 +1,184 @@
+package commands
+
+import (
+	"crypto/tls"
+	"fmt"
+	"github.com/argoproj/argo-workflows/v3/cmd/argo/commands/client"
+	"github.com/argoproj/argo-workflows/v3/pkg/apiclient"
+	workflowpkg "github.com/argoproj/argo-workflows/v3/pkg/apiclient/workflow"
+	"github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1"
+	"github.com/spf13/cobra"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"regexp"
+)
+
+type copyArtifactOps struct {
+	namespace    string // --namespace
+	workflowName string // --workflow-name
+	nodeId       string // --node-id
+	templateName string // --template-name
+	artifactName string // --artifact-name
+	artifactType string // --artifact-type
+}
+
+func NewCopyArtifactCommand() *cobra.Command {
```

Comment:
I'd probably call this func `NewCpCommand` and the file `cp.go`

Reply:
Updated naming